src/components/toast/components/close-button/close-button.module.css

```diff
@@ -0,0 +1,49 @@
+.closeButton {
+  composes: g-focus-ring-from-box-shadow from global;
+
+  --background-color: transparent;
+  --border-color: transparent;
+  --color: var(--token-color-foreground-faint);
+  --size: 24px;
```

Comment:
🙌🏻 Great idea to build the larger size into the element rather than just the `::before` pseudoelement!